lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+import NxBinaryDonutChart from '../NxBinaryDonutChart';
+
+describe('NxBinaryDonutChart', function() {
+  it('renders an svg with the expected properties', function() {
+    const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, { percent: 90 });
```

Comment:
This isn't really the right/most convenient way to use `getShallowComponent`.  See `NxCloseButton.test.tsx` for a simple set of examples.

Reply:
> This isn't really the right/most convenient way to use `getShallowComponent`. See `NxCloseButton.test.tsx` for a simple set of examples.

Done 76104c42b68ca9b5b18665b3149a349594ed02e3